src/common.ts

```diff
@@ -370,3 +369,14 @@ exports.validateParams = function(params, expectedParams, optionalParams, option
 
   return true;
 };
+
+export = {
+  Environments,
+  setNetwork,
```

Comment:
Exports can be simplified to `export Environments = {...`, `export setNetwork = function ...`. No need for export object